cmd/kubelet/app/options/options.go

```diff
@@ -512,6 +501,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 
 	fs.Int32Var(&c.NodeStatusMaxImages, "node-status-max-images", c.NodeStatusMaxImages, "The maximum number of images to report in Node.Status.Images. If -1 is specified, no cap will be applied.")
 	fs.BoolVar(&c.KernelMemcgNotification, "kernel-memcg-notification", c.KernelMemcgNotification, "If enabled, the kubelet will integrate with the kernel memcg notification to determine if memory eviction thresholds are crossed rather than polling.")
+	fs.BoolVar(&c.ExitOnLockContention, "exit-on-lock-contention", c.ExitOnLockContention, "Whether kubelet should exit upon lock-file contention.")
+	fs.StringVar(&c.LockFilePath, "lock-file", c.LockFilePath, "The path to file for kubelet to use as a lock file.")
```

Comment:
I am trying to chase back why the flag was classified as Alpha, it seems elevating it directly to GA without reference to the feature KEP for an Alpha thing to get promoted is something we want to clarify.

Reply:
I can't find a KEP for this, it predates the current KEP process. I found the design doc in the community repo: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cluster-lifecycle/self-hosted-kubernetes.md

The PR this got marked as alpha was in 2017: https://github.com/kubernetes/kubernetes/pull/55562